frontend/pages/policies/ManagePoliciesPage/components/CalendarEventPreviewModal/CalendarEventPreviewModal.tsx

```diff
@@ -63,7 +63,8 @@ const CalendarEventPreviewModal = ({
             </div>
             <div className={`${baseClass}__preview-info__text`}>
               {orgName} reserved this time to make some changes to your work
-              computer.
+              computer (
+              {showGenericPreview ? "ANNAS-DELL" : "Anna's MacBook Pro"}).
```

Comment:
What's this change about?

Reply:
slightly different copy in the [Figma](https://www.figma.com/design/ShDF2ZI9jFOwT1LMzMxYTv/%2319281-Add-computer's-name-to-maintenance-window?m=dev&node-id=1303-2404) for these two cases. Worth checking, @rachaelshaw was that intended?

Reply:
@jacobshandling I think we should actually keep it as "Anna's MacBook Pro" for everything. I had changed it in Figma to see how it would look with less pretty display names, but we should keep the more readable one for the preview! Sorry about that, I'll update the Figma.

 @lucasmrod thanks for catching this!